Setup/Installer/Windows/TeamTalk5.iss

```diff
@@ -59,12 +59,10 @@ Source: "C:\tt5dist\TeamTalk5_x64\TeamTalk5\Server\tt5svc.exe"; DestDir: "{app}"
 Source: "C:\tt5dist\TeamTalk5_x64\TeamTalk5\Server\tt5srv.exe"; DestDir: "{app}"; Components: server; Flags: ignoreversion; Check: Is64BitInstallMode;
 
 ;cfg files
-Source: "C:\tt5dist\TeamTalk5_x86\TeamTalk5\Setup\Client\Windows\TeamTalk5.ini"; DestDir: "{app}"; DestName: "TeamTalk5.ini.default"; Components: client; Flags: ignoreversion;

```

Comment:
If we remove TeamTalk5.ini.default then a user cannot reset the configuration. You can reset in the Help menu.

Reply:
I guess TeamTalk5.ini should just be empty

Reply:
I no longer remembered this possibility in the help menu. I restored the ini files by emptying them